### ISSUE TITLE
Help text on Ocp_Apim_Key_AccessToken is not updated

### ISSUE THREAD

**Issue by flinkgutt** [vippsas/vipps-woocommerce#13]
Vipps has put into production a system where you use a single key for both AccessToken and eCommerce subscription key. 
However the old system with two keys are still in use by the wast majority of users.

The help text does not reflect this, currently it's:
```
'Ocp_Apim_Key_AccessToken' => array(
    'title' => __('Subscription key for Access Token', 'woo-vipps'),
    'label'       => __( 'Subscription key for Access Token', 'woo-vipps' ),
    'type'        => 'password',
    'description' => __('The Primary key for the Access Token subscription from your profile on the developer portal','woo-vipps'),
    'default'     => '',
),
'Ocp_Apim_Key_eCommerce' => array(
    'title' => __('Subscription key for eCommerce', 'woo-vipps'),
    'label'       => __( 'Subscription key for eCommerce', 'woo-vipps' ),
    'type'        => 'password',
    'description' => __('The Primary key for the eCommerce API subscription from your profile on the developer portal','woo-vipps'),
    'default'     => '',
),
```

I suggest the text be something along the linse of:
'The Primary key for the Access Token subscription OR the Primary key for your "eCommerce-Services-API", both from your profile on the developer portal'

**Comment by iverok** [comment]
Fix in master branch, will release on monday.

**Comment by iverok** [comment]
Fixed in 1.1.15
